src/scripts/main.js

```diff
@@ -1,3 +1,200 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const elements = tableBody.querySelectorAll('tr');
+const users = [];
+
+tableHead.addEventListener('click', e => {
+  const element = e.target;
+
+  if (element.tagName === 'TH') {
+    if (!element.getAttribute('data-sort')) {
+      const dataSorts = document.querySelectorAll('[data-sort]');
+
+      dataSorts.forEach(dataSort => {
+        dataSort.removeAttribute('data-sort');
+      });
+    }
+
+    if (element.getAttribute('data-sort') === 'ASC') {
+      sortTableDESC(users, element.innerText);
+      createTable(users);
+      element.setAttribute('data-sort', 'DESC');
+    } else {
+      sortTableASC(users, element.innerText);
+      createTable(users);
+      element.setAttribute('data-sort', 'ASC');
+    }
+  }
+});
+
+elements.forEach(tr => {
+  users.push(
+    {
+      Name: tr.children[0].innerText,
+      Position: tr.children[1].innerText,
+      Office: tr.children[2].innerText,
+      Age: Number(tr.children[3].innerText),
+      Salary: Number(tr.children[4].innerText.replace(/[$,]/g, '')),
+    }
+  );
+});
+
+const sortTableASC = (table, value) => {
+  table.sort((a, b) => {
+    if (typeof a[value] === 'number') {
+      return a[value] - b[value];
+    } else if (typeof a[value] === 'string') {
+      return a[value].localeCompare(b[value]);
+    }
+
+    throw new Error('wrong typeof value');
+  });
+};
+
+const sortTableDESC = (table, value) => {
+  table.sort((a, b) => {
+    if (typeof a[value] === 'number') {
+      return b[value] - a[value];
+    } else if (typeof a[value] === 'string') {
+      return b[value].localeCompare(a[value]);
+    }
+
+    throw new Error('wrong typeof value');
+  });
+};
+
+const createTable = (table) => {
+  tableBody.innerHTML = '';
+
+  table.forEach(element => {
+    const tableElement = document.createElement('tr');
+    const salary = element.Salary.toLocaleString('en-US', {
+      style: 'currency',
+      currency: 'USD',
+      minimumFractionDigits: 0,
+      maximumFractionDigits: 0,
+    });
+
+    tableElement.innerHTML = (
+      `<td>${element.Name}</td>
+      <td>${element.Position}</td>
+      <td>${element.Office}</td>
+      <td>${element.Age}</td>
+      <td>${salary}</td>`
+    );
+
+    tableBody.appendChild(tableElement);
+  });
+};
+
+elements.forEach(element => {
+  element.addEventListener('click', () => {
+    const actives = document.querySelectorAll('.active');
+
+    actives.forEach(active => {
+      active.classList.remove('active');
+    });
+
+    element.classList.add('active');
+  });
+});
+
+const createForm = () => {
+  const newForm = document.createElement('form');
+
+  newForm.classList.add('new-employee-form');
```

Comment:
Instead of disabling eslint `max-len`, consider breaking the long string into multiple lines, this shouldn't break anything.